x-pack/plugins/triggers_actions_ui/public/plugin.ts

```diff
@@ -186,16 +195,22 @@ export class Plugin
       actionTypeRegistry: this.actionTypeRegistry,
     });
 
+    registerAlertsTable({
```

Comment:
Let's only do this if the experimental flag is set (since that's the only time the table should be visible)